statistics/ddl.go

```diff
@@ -100,12 +100,12 @@ func (h *Handle) insertColStats2KV(tableID int64, colInfo *model.ColumnInfo) err
 		if err != nil {
 			return errors.Trace(err)
 		}
-		var row types.Row
-		row, err = rs[0].Next(ctx)
+		chk := rs[0].NewChunk()
+		err = rs[0].NextChunk(ctx, chk)
 		if err != nil {
 			return errors.Trace(err)
 		}
-		count := row.GetInt64(0)
+		count := chk.NumRows()
```

Comment:
`NumRows` is the number of rows in this chunk while `count` is the first value in the first row.

Reply:
I still haven't found how to convert it, I also need some time to understand.